### ISSUE TITLE
How do I change music in fnf apk

### ISSUE THREAD

**Issue by GDHatdaug1992** [luckydog7/Funkin-android#1018]
If I use apk editor and install the game the song song does not play

**Comment by NeonNotFound** [comment]
> If I use apk editor and install the game the song song does not play

You need to make it an ogg file

**Comment by GDHatdaug1992** [comment]
yea I did that and still it didn't work

**Comment by Vicdragon** [comment]
> yea I did that and still it didn't work

Did you convert to a ogg file?

**Comment by CreatorOlixter** [comment]
Yeah, Don't rename the file, you should use an online con verter or a convertir app

**Comment by REAL-DoritosMAN** [comment]
If the song won't work it's probably because:
Not .ogg file (use online converter for that)
Song is too long (not sure about limits)

I would use apk editor because its easier
